### ISSUE TITLE
0036-BRIE-event_queue documentations inc. describing how confirmations/voting works

### ISSUE THREAD

**Issue by gordsport** [vegaprotocol/specs#893]
Update 0036-BRIE-event_queue as it does not describe how confirmations/voting works

As part of the gap analysis for the open issue in [asana](https://app.asana.com/0/1201666015335982/1201802898848442) its required that we have documentation covering the event types and the event triggers for these.

Ideally we will have this before starting:
- https://github.com/vegaprotocol/system-tests/issues/467

**Comment by gordsport** [comment]
It has been request that when this issue is picked up that we use the term EEF in the spec as that is what its widely been called and it would be easier to link discussions to the spec

**Comment by gordsport** [comment]
closing as a duplicate of: 
- https://github.com/vegaprotocol/specs/issues/1173
